src/components/Disclosure/DisclosureSummary/DisclosureSummary.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {ArrowToggle} from '../../ArrowToggle';
+import {DisclosureSize, b} from '../Disclosure';
```

Comment:
- Here the same about circular denepdency
- It would be better to import types with keyword `type`